src/api/dashboard.ts

```diff
@@ -1,8 +1,13 @@
-export default function() {
-
-	return {
-		getDashboardData: async () => {
-
-		},
-	}
+export default function () {
+  return {
+    getData: async() => {
+      let url = 'https://random-data-api.com/api/device/random_device?size=20';
```

Comment:
General feedback, use `const` instead of `let` whenever a variable's value is not going to change